src/doc/DocAnnotator.js

```diff
@@ -533,6 +533,16 @@ class DocAnnotator extends Annotator {
      * @return {void}
      */
     onSelectionChange(event: Event) {
+        const selection = window.getSelection();
+        const selectionNode = selection.anchorNode;
+        // Check parent node in IE, which will return a text node instead of a div when accessing the anchorNode
```

Comment:
Could you clarify this comment a little more so we know what this case is for in the future?